packages/web-components/@groupby/elements-search-box/stories/index.ts

```diff
@@ -11,6 +18,20 @@ const updateTextEvent = [
   },
 ];
 
+const eventsToListen = [
+  SEARCHBOX_CLEAR,
+  SEARCHBOX_CLICK,
+  SEARCHBOX_INPUT,
+  SEARCH_REQUEST,
+  UPDATE_SEARCH_TERM
+];
+
+eventsToListen.forEach((eventName) => {
```

Comment:
If we never use `eventsToListen` again, you can simply inline it and invoke the `forEach` directly on the array, unless you think the way you have it is much clearer.

Reply:
@jmbtutor I was actually thinking that we would use this function again in other stories and maybe I should move it to some appropriate utils place maybe in the storybook config folder?

Reply:
@ssbelle I also think it's likely we'll re-use it, and so a more global function is definitely useful (as you have implemented). Another consideration is whether this function should exist in your library so we can simply import it. It's not necessary right away, but it's worth considering if your library's purpose extends to that.

Reply:
@DanielMoniz-GBI I currently moved it to the storybook common js file with other utils functions we use throughout multiple stories. Does that work?

Reply:
@DanielMoniz-GBI yes I had thought about moving that functionality into the package, I can definitely do that now or later

Reply:
@ssbelle I'm guessing it'll happen later!